src/SmartComponents/DriftPage/DriftTable/ComparisonHeader/SystemHeaderCellContent.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import RemoveSystemCell from './RemoveSystemCell';
+import DriftTooltip from '../../../DriftTooltip/DriftTooltip';
+import HistoricalProfilesPopover from '../../../HistoricalProfilesPopover/HistoricalProfilesPopover';
+import { BlueprintIcon, ClockIcon, ExclamationTriangleIcon, ServerIcon } from '@patternfly/react-icons';
+import ReferenceSelector from '../ReferenceSelector/ReferenceSelector';
+import helpers from '../../../helpers';
+
+const SystemHeaderCellContent = ({
+    fetchCompare,
+    item,
+    permissions,
+    referenceId,
+    removeSystemFunc,
+    selectHistoricProfiles,
+    systemIds,
+    updateReferenceId
+}) => {
+    const selectedBaselineIds = useSelector(({ baselinesTableState }) => baselinesTableState.comparisonTable.selectedBaselineIds);
+    const getItemType = (itemType) => {
+        if (itemType === 'system') {
+            return 'System';
+        } else if (itemType === 'baseline') {
+            return 'Baseline';
+        } else if (itemType === 'historical-system-profile') {
+            return 'Historical System';
+        }
+    };
+
+    const getItemIcon = (itemType) => {
+        if (itemType === 'system') {
+            return <ServerIcon/>;
+        } else if (itemType === 'baseline') {
+            return <BlueprintIcon/>;
+        } else if (itemType === 'historical-system-profile') {
+            return <ClockIcon />;
+        }
+    };
```

Comment:
This could be in an object outside the component function. Like:

```js

const itemTypeIcons = {
  system: {
    type: 'System'
    Icon: ServerIcon
  }
  ...
};

...

const {type: itemType, Icon: ItemIcon} = itemTypeIcons[item.type];
....


<DriftTooltip
  content={ itemType }
  body={ <ItemIcon /> }
/>
...
```